cvat/apps/dashboard/templates/dashboard/dashboard.html

```diff
@@ -143,6 +143,33 @@
                             <input type="checkbox" id="dashboardCustomQuality" title="Custom image quality"/>
                         </td>
                     </tr>
+                    <tr>
+                        <td>
+                            <label class="regular h2"> Start Frame </label>
+                        </td>
+                        <td>
+                            <input type="number" id="dashboardStartFrame" class="regular" style="width: 4.5em;" min="0" value=0 disabled=true/>
+                            <input type="checkbox" id="dashboardCustomStart" title="Custom start frame"/>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <label class="regular h2"> Stop Frame </label>
+                        </td>
+                        <td>
+                            <input type="number" id="dashboardStopFrame" class="regular" style="width: 4.5em;" min="0" value=0 disabled=true/>
+                            <input type="checkbox" id="dashboardCustomStop" title="Custom stop frame"/>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>
+                            <label class="regular h2"> Frame Filter </label>
+                        </td>
+                        <td>
+                            <input type="text" id="dashboardFrameFilter" class="regular" style="width: 4.5em;" disabled=true/>
```

Comment:
Need a hint here about possible values (at the moment only one value "step=K")

Reply:
Hint message added